topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,160 @@
+
+contributors:
+- deeptivarshney
+
+---
+
+# Introduction
+
+<!-- This is a comment. -->
+
+MAdLand is a collection of fully sequenced plant and algal genomes, with a focus on non-seed plants and streptophyte algae. For comparison, it includes genomes from fungi, animals, the SAR group, bacteria, and archaea. It is developed and maintained by the [Rensing lab](http://plantco.de). The species are abbreviated by a 5 letter code, which consists of the first three letters of the genus and the first two of the species name, e.g. CHABR for Chara braunii. We add the gene ID to that and additional shortcuts, like whether it is plastome encoded (pt) or transcriptome-based (tr, in cases when no genome is available yet).
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Access the Galaxy server
+
+> <hands-on-title>Log in or register</hands-on-title>
+>
+> 1. Open your favorite browser (Chrome/Chromium, Safari, or Firefox, but not Internet Explorer/Edge!)
+> 2. Browse to [Galaxy](https://usegalaxy.eu) 
+> 3. Choose *Login or Register* from the navigation bar at the top of the page
+> 4. If you have previously registered an account with this particular instance of Galaxy (user accounts are *not* shared between public servers!), proceed by logging in with your registered *public name*, or email address, and your password.
+>
+>    If you need to create a new account, click on *Register here* instead.
+>
+
+# Upload data on Galaxy
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a proper name
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>    {% snippet faqs/galaxy/histories_rename.md %}
+
+
+# Perform NCBI Blast+ on Galaxy 
+
+> After successfully logging in to the Galaxy server, Go to the [NCBI-Blast+](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) tool.  
+> Since MAdLandDB is the collection of protein sequences, You can perform [BLASTp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) and [BLASTx](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastx_wrapper/2.10.1+galaxy2) tools.
+
+>
+> <hands-on-title> Similarity search against MAdLand Database </hands-on-title>
+>
+> 1. {% icon tool %} As input file, select the protein sequences 
+> 2. Choose the protein BLAST database *MAdLandDB* with the following parameters:
+>    - In *"input Options"*:
+>        - {% icon param-file %} *"Protein query sequence(s)"*: `output` (Input dataset)
+>        - *"Subject database/sequences"*: `Locally installed BLAST database`
+>        - *"Protein BLAST database"*: `MAdLandDB`
+>    - In *"Output Options"*:
+>       - *"Output format"*: `Tabular (extended 25 columns)` (optional)
```

Comment:
```suggestion
> <hands-on-title> Similarity search against MAdLand Database </hands-on-title>
>
> 1. {% tool [NCBI BLAST+ blastp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) %} with the following parameters:
>    -  *"Protein query sequence(s)"*:  `Aminoacid input sequence`
>    - *"Subject database/sequences"*:  `Locally installed BLAST database`
>    - *"Protein BLAST database"*: `MadLandDB (Genome zoo) plant and algal genomes with a focus on non-seed plants and streptophyte algae (22 Dec 2022)`
>    - *"Type of BLAST"*:  `blastp - Traditional BLASTP to compare a protein query to a protein database`
>    - *"Set expectation value cutoff"*: `0.001`
>    - *"Output format"*: 
>    - In *"Output Options"*: `Tabular (extended 25 columns)` 
```

Reply:
Since the pipeline described in this tutorial is not very large, I would suggest you go deeper in the advanced options and explain them a little bit; for example, depending on the input sequences, it is recommended to use a specific score matrix. Also, wouldn't be possible to use DIAMOND instead BLASTp? As DIAMOND is supposed to be much faster.